src/garrysmod/gamemodes/zombiesurvival/gamemode/sv_database.lua

```diff
@@ -5,27 +5,29 @@ local password = CreateConVar("postgres_password", "zombiesurvival", bit.bor(FCV
 local user = CreateConVar("postgres_user", "zombiesurvival", bit.bor(FCVAR_PROTECTED), "")
 local host = CreateConVar("postgres_host", "database", bit.bor(FCVAR_PROTECTED), "")
 
-GM.Database = GM.Database or {}
-GM.Database._Connected = false
-GM.Database._Queue = {}
+GM.Database = GAMEMODE and GAMEMODE.Database or {
+    _Connection = nil,
+    _Connected = false,
+    _Queue = {}
+}
```

Comment:
This is to fix issues with autorefresh. Once the Lua environment is active, you have to reference the gamemode table with `GAMEMODE`, not `GM`. So, this was always zapping the table, thus zapping the connection object and queue.